packages/common/types/fast-xml-parser.d.ts

```diff
@@ -0,0 +1 @@
+declare module "fast-xml-parser";
```

Comment:
for my own edification, why do we have to do this?

Reply:
I think b/c that library doesn't provide any types (at least at the version we are using). The build fails w/o it and shows a message saying to add this line if the types aren't available.